boards/arm/mimxrt1050_evk/Kconfig.defconfig

```diff
@@ -36,4 +36,11 @@ config UART_MCUX_LPUART_1
 
 endif # UART_MCUX_LPUART
 
+if CODE_HYPERFLASH || CODE_QSPI
+
+config TEXT_SECTION_OFFSET
```

Comment:
Can we add a comment either here or in commit message about why the offset.  Is this space reserved for IVT?

Reply:
Yes. I added a comment here.